### PR TITLE
Results relax sharedmem and simplification

### PR DESCRIPTION
**Pull request by gisjedi**
Addresses issues #68 #74 #75 #77

### CODE REVIEW

File: spec/sections/standard.adoc [hunk 1]
```diff
@@ -127,14 +128,14 @@ time. Reference <<environment-variables>> for clarification on what the implemen
 objects and MAY contain any number of elements. There is no other standard restriction on the array size.
 
 .Scalar Elements
-The Scalar objects MAY include any arbitrary custom resource name, but there are reserved resources `cpu`, `disk` and
-`mem` that have special meaning and SHOULD be populated by all Seed compliant images.
+The Scalar objects MAY include any arbitrary custom resource name, but there are reserved resources `cpu`, `disk`,
+`mem` and `sharedMem` that have special meaning and SHOULD be populated by all Seed compliant images.
```
Comment (dfaller) [inline spec/sections/standard.adoc]: sharedMem will only be applicable to some algorithms, so I don't think providing it is a "SHOULD" for all Seed images.

File: spec/sections/standard.adoc [hunk 2]
```diff
@@ -176,14 +177,14 @@ input file size for scaling memory requirement of job.
 The Interface object is the primary member that describes the command arguments, environment variables,
 mounts, settings, inputs and outputs defined for a job.
 
-* The Interface object MAY have a member with the name `cmd`. The member's value MUST be a string specifying the
+* The Interface object MAY have a member with the name `command`. The member's value MUST be a string specifying the
```
Comment (dfaller) [inline spec/sections/standard.adoc:183]: Should the command be renamed to indicate that it is the command arguments?

`The Seed command member can be treated as analogous to the Docker CMD statement.`
Is this true? I thought we were expecting Seed images to have a set entrypoint (which would be the real command) and we were just passing arguments.

File: spec/sections/standard.adoc [hunk 3]
```diff
@@ -196,28 +197,28 @@ The following annotated snippet provides quick reference to the use of Interface
...
+<3> Optional <<outputs-section>>. This is the means to capture results from the job container.
 <4> Optional <<mounts-section>>. This defines any directories that need to be mounted into the job container.
 <5> Optional <<settings-section>>. This defines any environment specific settings needed at run time.
 
-[[inputdata-section, InputData Object]]
-====== InputData Object
-The InputData object is the member responsible for indicating all mutable content available to the Seed image at
+[[inputs-section, Inputs Object]]
+====== Inputs Object
+The Inputs object is the member responsible for indicating all immutable content available to the Seed image at
```
Comment (dfaller) [inline spec/sections/standard.adoc]: I wonder if this should be reworded to talk about input data. Mounts are technically another way to provide "immutable content available to the Seed image at runtime".

File: spec/sections/usage.adoc [hunk 4]
```diff
@@ -62,9 +62,9 @@ is important to note this is specifically a Bash shell feature and is not presen
 ==== Output Data Permissions
 All output data generated by a Seed compliant algorithm MUST be placed within the location identified by OUTPUT_DIR
 environment variable. This location SHALL be made available with permissions to be writable to the algorithm by the
-execution framework. Algorithm developers MUST ensure any `interface.outputData.files` member elements, as well as the
-optional `results_manifest.json` file, are marked with read permissions at minimum. This MAY be done with the following
-sample command: `chmod +x yourfile.txt`
+execution framework. Algorithm developers MUST ensure any `interface.outputs.files` member elements, as well as the
+optional `seed.output.manifest.json` file, are marked with read permissions at minimum. This MAY be done with the
+following sample command: `chmod +x yourfile.txt`
```
Comment (dfaller) [inline spec/sections/usage.adoc]: Don't you mean `chmod +r yourfile.txt`?

**Review by dfaller — CHANGES_REQUESTED** [review]
I added some discussion points.

File: spec/sections/usage.adoc (hunk 4 above)
Comment (gisjedi) [reply spec/sections/usage.adoc]: Yes, I do :-)

File: spec/sections/standard.adoc (hunk 2 above)
Comment (gisjedi) [reply spec/sections/standard.adoc:183]: It is definitely true and you are correct we do expect them to set an ENTRYPOINT. The Docker CMD always is additional arguments to the ENTRYPOINT command. It is just usually the default ENTRYPOINT is /bin/sh or the like which results in CMD appearing to be the full executable AND args.

**Review by dfaller — APPROVED** [review]
LGTM

**Pull request merged**